media/js/firefox/nothing-personal/random-answers.es6.js

```diff
@@ -0,0 +1,43 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+let randomizerButton;
+let randomOutput;
+let answers;
+let mutableAnswers;
+
+function getRandomAnswer(answers) {
+    // set a mutable array we can work through, reset it when empty
+    if (mutableAnswers === undefined || mutableAnswers.length === 0) {
+        mutableAnswers = [...answers];
+    }
+    const answerIndex = Math.floor(Math.random() * mutableAnswers.length);
+    // mutate array to remove answer so we can't repeat it, return answer
+    return mutableAnswers.splice(answerIndex, 1);
+}
+
+function setRandomAnswer(container, answers) {
+    container.textContent = getRandomAnswer(answers);
```

Comment:
**Analytics:** An event for this will be covered by adding `data-cta-text` to the button that triggers it so no action is needed here.